swagger.yaml

```diff
@@ -1233,4 +1309,59 @@ components:
       enum:
         - CREATE
         - UPDATE
-        - DELETE
\ No newline at end of file
+        - DELETE
+
+    DeviceSurveyData:
+      type: object
+      description: Survey object that is sent to devices and contains only necessary info for display
+      properties:
+        id:
```

Comment:
id, pages, surveyId, deviceId and status could be marked as required and all should be marked as readOnly (this is readOnly entity)